### PR TITLE
Added godeps and go report card badge to readme

### PR DESCRIPTION
**Pull request by myles-mcdonnell**
Hi Kyle

Don't know if you might find Godeps a useful addition?  The report card badge is a nice touch too and you were A+ with no amends 👍  apologies if I'm telling you how to suck eggs.

Thanks
Myles

### CODE REVIEW

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+	"ImportPath": "github.com/myles-mcdonnell/go-monitor",
```
Comment (ksred) [inline Godeps/Godeps.json]: Can this be changed to the correct github url please?

**Comment by ksred** [comment]
This is awesome - thanks!

File: Godeps/Godeps.json (hunk 1 above)
Comment (myles-mcdonnell) [reply Godeps/Godeps.json]: Doh! well spotted..

**Comment by ksred** [comment]
Awesome!

**Pull request merged**